roles/reverseproxy/tasks/server.yml

```
@@ -0,0 +1,39 @@
+---
+- name: install nginx
+  ansible.builtin.apt: pkg=nginx state=present
```

```suggestion
  ansible.builtin.apt:
    pkg: nginx
    state: present
```
Might be personal preference although this blog article (https://www.ansible.com/blog/ansible-best-practices-essentials) agrees with me. Both will work.

---

I tend to do it out of habit for single packages/to keep the playbooks shorter if the arguments are simple, but I'm happy to follow best practices (and the arguments for it make plenty of sense)